web-frontend/src/pages/TransferCoins/SendCoins/Send.tsx

```diff
@@ -0,0 +1,176 @@
+import { Balance, Button, Input } from '@massalabs/react-ui-kit';
+import { FiUsers } from 'react-icons/fi';
+import useResource from '../../../custom/api/useResource';
+import { AccountObject } from '../../../models/AccountModel';
+import { useLocation } from 'react-router-dom';
+import { parseForm } from '../../../utils/parseForm';
+import { useState } from 'react';
+
+// remove padding from Balance component
```

Comment:
You can use `customClass="pl-0"` in your <Balance> component to fix in the left

However, pull the ui-kit and update in the project